src/main/scala/me/rexim/issuestant/splittree/SplitIssueForest.scala

```diff
@@ -0,0 +1,37 @@
+package me.rexim.issuestant.splittree
+
+import me.rexim.issuestant.github._
+
+class SplitIssueForest(issues: List[Issue]) {
+  private def sortForest(forest: List[SplitIssueTree]): List[SplitIssueTree] =
+    forest.sortBy(_.number).map { tree =>
+      tree.copy(children = sortForest(tree.children))
+    }
+
+  private lazy val forest: List[SplitIssueTree] = sortForest {
+    val initialForestMap: Map[Int, SplitIssueTree] =
+      issues.map { issue =>
+        issue.number -> SplitIssueTree.fromIssue(issue)
+      }.toMap
+
+    val forestRootNumbers: Set[Int] =
+      issues
```

Comment:
No
1. `parentNumber` comes from implicit class `IssueWithParent`. Can't be pattern matched that easily.
2. `Issue` is a data structure that reflects GitHub API response. It will be changed in the future quite frequently and I don't wanna change this specific pattern every time.